kops/cloud-platforms-test.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: null
+spec:
+  api:
+    loadBalancer:
+      type: Public
+  additionalPolicies:
+    node: |
+      [
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ChangeResourceRecordSets"
+          ],
+          "Resource": [
+            "arn:aws:route53:::hostedzone/Z1N3S2UZYKP74S"
+          ]
+        },
+        {
+          "Effect": "Allow",
+          "Action": [
+            "route53:ListHostedZones",
+            "route53:ListResourceRecordSets"
+          ],
+          "Resource": [
+            "*"
+          ]
+        }
+      ]
+  authorization:
+    rbac: {}
+  channel: stable
+  cloudProvider: aws
+  iam:
+    allowContainerRegistry: true
+    legacy: false
+  kubeAPIServer:
+    oidcClientID: 7ix4YkYwQe8n7j9qTrdAKeMz26eJTB2x
+    oidcIssuerURL: https://moj-cloud-platforms-dev.eu.auth0.com/
+    oidcUsernameClaim: nickname
+    oidcGroupsClaim: https://k8s.integration.dsd.io/groups
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  kubernetesVersion: 1.8.6
+  networking:
+    calico: {}
+  nonMasqueradeCIDR: 100.64.0.0/10
+  sshAccess:
+  - 0.0.0.0/0
+
+---
+
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: null
+  name: master-eu-west-1a
+spec:
+  image: kope.io/k8s-1.8-debian-jessie-amd64-hvm-ebs-2018-01-14
+  machineType: t2.medium
```

Comment:
We should probably also have instance types and numbers as arguments in the cluster creation script, so we can deploy smaller clusters for testing and individual dev clusters

Reply:
or we can have a testing branch of this repo with the smaller instance size, and link that branch to a create-cluster-test-pipeline job that will run that against test cluster for test purposes before merging to master (prod)

Reply:
if we put instance type and numbers as arguments for the cluster creation script, its not going to live in code, nor in git.
If we put it in the pipeline, then we will need one pipeline per cluster creating duplication.
I will suggest leaving this for now as it is.
the cluster yaml specification fill will have the size and number of instances for each cluster.

Reply:
Yep, agreed - we can pull it out later, or pull the `InstanceGroup` specs into separate YML files later if it proves to be a problem.